### PR TITLE
new uplink item for chefs, wendigo meat

### PR DESCRIPTION
**Pull request by Feldnos**
Costs 8 TC for non chefs, 2 tc for chefs

force feed it to any monkeyman to make a wendigo and release it in maintence somewhere with a whole bunch of monkeymen, watch greyshits and redshirts alike flee in terror
first pr lemme know if i need to fix anything
🆑 
 * rscadd: new uplink items for syndicate chefs

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
needs a changelog

**Comment by boy2mantwicethefam** [comment]
I should look into wendigos tomorrow because they've been pretty fucked up for a while, a lot of their unique behavior is relegated to AI-only behavior and no player behavior

**Comment by boy2mantwicethefam** [comment]
Oh yeah, also the wendigo transformation means the original person is gone. It's okay though, the game is meant to be dangerous

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -835,6 +835,14 @@ var/list/uplink_items = list()
 	discounted_cost = 2
 	jobs_with_discount = list("Chef")
 
+/datum/uplink_item/jobspecific/service/specialsauce
```
Comment (GreatBigFailure) [inline code/datums/uplink_item.dm]: Forgot to change this. Should be
/datum/uplink_item/jobspecific/service/wendigomeat

**Review by GreatBigFailure — CHANGES_REQUESTED** [review]

**Comment by Feldnos** [comment]
> Oh yeah, also the wendigo transformation means the original person is gone. It's okay though, the game is meant to be dangerous

i plan to make em like zombies but with how wendigos are right now putting a person in them is both broken and serves no purpose, since you cannot evolve while your ckey is in the body (no verb to take "bites" or consume people" and breaking bones with one hit going at super speed
for a person to transform they'd basically have to ignore every single message while not getting to anti tox

**Comment by GreatBigFailure** [comment]
I think 2TC is a bit too cheap, especially considering it's technically renewable and how strong wendigos can get. It could definitely lead to some interesting rounds though. A few super strong mobs is probably more interesting of a mobloose than a million weaker ones.

**Comment by Feldnos** [comment]
> I think 2TC is a bit too cheap, especially considering it's technically renewable and how strong wendigos can get. It could definitely lead to some interesting rounds though. A few super strong mobs is probably more interesting of a mobloose than a million weaker ones.

what price do you think would be better for it?

**Comment by GreatBigFailure** [comment]
Each wendigo drops 1 meat so it's not like 1 can be farmed for an infinite supply, I guess that's not much of a concern. 
Maybe 3 to 6. 6 would let them get a bundle as well, or a master trainer's belt so they can feed the wendigos, if that works? 
I'm not sure how strong each wendigo is at each tier so maybe I'm overthinking it.  If it was 6 then spending your whole TC stash on 3 mobs isn't great, though wendigo meat can be bitten a bunch of times I think.

**Comment by Feldnos** [comment]
> Each wendigo drops 1 meat so it's not like 1 can be farmed for an infinite supply, I guess that's not much of a concern.
> Maybe 3 to 6. 6 would let them get a bundle as well, or a master trainer's belt so they can feed the wendigos, if that works?
> I'm not sure how strong each wendigo is at each tier so maybe I'm overthinking it. If it was 6 then spending your whole TC stash on 3 mobs isn't great, though wendigo meat can be bitten a bunch of times I think.

could make it like the vox box meat cloner, wendigo meat comes in a box, one meat, 4 human cubes ,3 lets it evolve to the next stage of the transformation, additional humans would have to be gotten manually, takes like 10 to make an alpha wendigo, although im not sure how wendigo transformation procs i have a feeling it procs on taking a bite and not actually finishing the meat, in which case i should definitely up the price

**Comment by Blithering** [comment]
I think wendigos might be blacklisted from the master trainer belt, they're not quite mechahitlers, but still miniboss enemies. Plus the sprite for the second state really needs updating at this point, I should get on that.

**Comment by zakanater19** [comment]
Do they spread it by attacking or no?

**Comment by Feldnos** [comment]
> Do they spread it by attacking or no?

they dont spread, no, but they evolve, much easier than zombies, and they have an okay ai that makes them somewhat of a threat for anyone not already armed to the teeth with riot armor or explosives

**Comment by SonixApache** [comment]
you fucked up the 🆑 
should be like this
```
:cl:
 * rscadd: new uplink items for syndicate chefs
```

**Comment by dunsel-hordon** [comment]
So explain wendingos? I thought they could also make more wendingos somehow?

**Comment by Blithering** [comment]
Wendigoes get stronger by eating people, evolving into a successively stronger form two times. They can't make more wendigoes but killing one and eating the meat passes the curse to you, turning you into a new wendigo NPC.

**Comment by SonixApache** [comment]
you still didn't do the :cl: right, just copypaste the entire thing

**Comment by DamianX** [comment]
Votes are split, also I hate it.

**Pull request closed without merging**